docs/installation/linux/debian.md

```diff
@@ -36,6 +36,26 @@ Docker is supported on the following versions of Debian:
 
      $ uname -r
 
+ Additionally, for users of Debian Wheezy, backports must be available. To enable backports in Wheezy:
+ 
+ 1. Log into your machine and open a terminal with `sudo` or `root` privileges.
+ 
+ 2. Open the `/etc/apt/sources.list.d/backports.list` file in your favorite editor.
+
+     If the file doesn't exist, create it.
+ 
+ 3. Remove any existing entries.
+ 
+ 4. Add an entry for backports on Debian Wheezy.
+
+     An example entry:
+
+    deb http://http.debian.net/debian wheezy-backports main
+
+ 5. Update package information:
+
+         $ apt-get update
```

Comment:
Yeah I'm not sure anyone has ever found the right balance. I feel it's
better to say in the first step you need to be root to do the following
than just prepend sudo to everything. That way unless they actually read
steps in order, they won't do privileged things in the wrong order. They'll
actually get a verbose error message (ie, you should be root).

In terms of getting support requests from thay, by this stage I think we
could expect support requests regardless from people like that. At least we
conveyed things in the clearest most educational way for those that did
choose to read it and read it in order.

On Thu, 26 May 2016, 11:18 Sebastiaan van Stijn, <notifications@github.com>
wrote:

> In docs/installation/linux/debian.md
> <https://github.com/docker/docker/pull/22999#discussion_r64722348>:
>
> > +
> > + 2. Open the `/etc/apt/sources.list.d/backports.list` file in your favorite editor.
> > +
> > +     If the file doesn't exist, create it.
> > +
> > + 3. Remove any existing entries.
> > +
> > + 4. Add an entry for backports on Debian Wheezy.
> > +
> > +     An example entry:
> > +
> > +    deb http://http.debian.net/debian wheezy-backports main
> > +
> > + 5. Update package information:
> > +
> > +         $ apt-get update
>
> @deed02392 <https://github.com/deed02392> Oh, you're right, it's only the
> last steps (below "install docker") that are using sudo. Perhaps we
> should add a "1. If you haven't already..." to that block.
>
> It's difficult finding the right balance here; for some people it's
> "obvious" you have be privileged, or use sudo. For others, we keep
> getting reports "it doesn't work!".
>
> Okay, let's keep out the sudo here; I'll do a follow up PR to make the
> last block consistent 👍
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/22999/files/5fa9b8db7fb04425f77ef97bfee826cd40259efd#r64722348>
>